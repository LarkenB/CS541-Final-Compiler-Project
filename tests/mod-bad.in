int x = 100;
float y = 10;
int z = x % y;