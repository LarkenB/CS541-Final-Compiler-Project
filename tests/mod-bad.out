target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-pc-linux-gnu"

define dso_local i32 @main() {
  %t1 = alloca i32
  store i32 100, i32* %t1
  %t2 = alloca float
  %t3 = sitofp i32 10 to float
  store float %t3, float* %t2
  ret i32 0
}
